### PR TITLE
add ecr retrieval failed page

### LINKED ISSUE #2851: "eCR retrieval failed" error page
## What needs to be done

When the user navigates from NBS to an ecr that does not exist display the page as illustrated in the figma link.

## Why it needs to be done

If an error occurs users need to be better informed about what the error is.

## Acceptance Criteria

- [ ] When a user tries to load an ecr that does not exist in the integrated viewer this page is displayed

## Design Notes

[Link to Figma design](https://www.figma.com/design/zAPyOEYwNtx3f2Y47aoq0G/eCR-Viewer-(Aug-Dec-2024)?node-id=7887-158827&t=DuIFSC9bvXyJtsYO-4)

### PR DESCRIPTION
**Pull request by lina-roth**
# PULL REQUEST

Screen Shot 2024-11-08 at 3 10 39 PM

## Summary
- Add ecr retrieval failed page 
- move not found error page to correct place

## Related Issue
Fixes #2851 

## Acceptance Criteria
- [ ] When a user tries to load an ecr that does not exist in the integrated viewer this page is displayed

## Checklist

- [ ] If this code affects the other scrum team, have they been notified? (In Slack, as reviewers, etc.)

[//]: # (PR title: Remember to name your PR descriptively!)

### CODE REVIEW

File: containers/ecr-viewer/src/app/page.tsx [hunk 1]
```diff
@@ -30,34 +30,32 @@ const HomePage = async ({
...
-                sortColumn={sortColumn}
-                sortDirection={sortDirection}
-                searchTerm={searchTerm}
-              />
-            </EcrPaginationWrapper>
-          </>
-        ) : (
-          <NotFound />
-        )}
+        <>
```
Comment (mcmcgrath13) [inline containers/ecr-viewer/src/app/page.tsx:37]: since the ternary was dropped, the empty tag wrapper can be as well (plus closing)
```suggestion
```

Comment (lina-roth) [reply containers/ecr-viewer/src/app/page.tsx]: fixed!

File: containers/ecr-viewer/src/app/view-data/retrieval-failed.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        <div className="text-semibold font-sans-md margin-top-1">
+          The eCR Viewer couldn't retrieve the associated eCR file
+        </div>
+        <div className="bg-info-lighter border border-info-light radius-md font-sans-md line-height-sans-4 padding-3 margin-top-2">
+          This is likely because the DIBBs pipeline hasn't processed this eCR.
+          <p />
+          <div className="margin-0">
+            <b>Contact support:</b> If the problem persists, please reach out to
+            your eCR coordinator
+            <br /> to troubleshoot the issue with the DIBBs team.
```
[the remark this answers is not in the document]
Comment (lina-roth) [reply containers/ecr-viewer/src/app/view-data/retrieval-failed.tsx:31]: id normally agree but this being a static page with content that is unlikely to change, I dont feel like using text-wrap over <br> is really much of a difference

[the remark this answers is not in the document]
Comment (mcmcgrath13) [reply containers/ecr-viewer/src/app/view-data/retrieval-failed.tsx:31]: how does it look on mobile?

[the remark this answers is not in the document]
Comment (lina-roth) [reply containers/ecr-viewer/src/app/view-data/retrieval-failed.tsx:31]: I need to double check but I dont think we are concerned with mobile for this app

[the remark this answers is not in the document]
Comment (lina-roth) [reply containers/ecr-viewer/src/app/view-data/retrieval-failed.tsx:31]: Screen Shot 2024-11-08 at 4 36 01 PM

[the remark this answers is not in the document]
Comment (mcmcgrath13) [reply containers/ecr-viewer/src/app/view-data/retrieval-failed.tsx:31]: thanks! I could see it being weird at some widths, but if we're not worried about mobile, those odds are way low and I agree it doesn't make much difference here

[the remark this answers is not in the document]
Comment (mcmcgrath13) [reply containers/ecr-viewer/src/app/view-data/retrieval-failed.tsx:31]: though I suppose things could get weird again if someone has different font sizes set on their settings? 🤔

[the remark this answers is not in the document]
Comment (lina-roth) [reply containers/ecr-viewer/src/app/view-data/retrieval-failed.tsx:31]: i asked Emma and she said "our assumption is that all users will be in a desktop/workplace setting"

**Review by mcmcgrath13 — APPROVED** [review]
thanks!

**Pull request merged**